src/CodingTheory.jl

```diff
@@ -91,7 +91,7 @@ export kronecker_product, Hamming_weight, weight, wt, Hamming_distance, distance
     is_valid_bipartition, extract_bipartition, is_Hermitian_self_orthogonal,
     row_supports, row_supports_symplectic, strongly_lower_triangular_reduction,
     residue_polynomial_to_circulant_matrix, group_algebra_element_to_circulant_matrix,
-    load_alist
+    load_alist, _non_pivot_cols
```

Comment:
Anything starting with a _ is a private function and should not be exported. To use it in the REPL, do `CodingTheory._non_pivot_cols`.